src/index.html

```diff
@@ -56,7 +56,6 @@
     <script src="../node_modules/d3/d3.js"></script>
     <script src="../node_modules/nvd3/build/nv.d3.js"></script>
     <script src="../node_modules/angular-nvd3/dist/angular-nvd3.js"></script>
-    <script src="../node_modules/save-svg-as-png/lib/saveSvgAsPng.js"></script>

```

Comment:
This is needed for the report when the QPPQ/FDCTM method is enabled

Reply:
weird , i didnt take that out

maybe my npm lib is old